documentation/src/docs/security/generating-tls-artifacts.md

```diff
@@ -0,0 +1,324 @@
+                -import -file <ca-cert-file-path> -storepass <server-jks-keystore-password>
+   
+   # Import the signed server certificate into the keystore.
+   $ keytool -keystore <server-jks-keystore-file-path> -alias <server-alias> -noprompt \
+                -import -file <signed-server-cert-file-path> -storepass <server-jks-keystore-password>
+   ```
+   
+   Example:
+   ```bash
+   $ keytool -keystore server.keystore.jks -alias CARoot -noprompt \
+             -import -file ca-cert.crt -storepass changeit
+
+   $ keytool -keystore server.keystore.jks -alias server -noprompt \
+                -import -file server-cert.crt -storepass changeit
+   
+   # Optionally, list the keystore and inspect the output
+   $ keytool -list -v -storepass changeit -keystore server.keystore.jks
+   ```
+
+5. Export the server certificate's private key. 
+   
+   This is a two-step process.
+   * First, convert the server's keystore in `.jks` format into `.p12` format.
+
+     ```bash
+     $ keytool -importkeystore \
+               -srckeystore <server-jks-keystore-file-path> \
+               -destkeystore <server-pkcs12-keystore-file-path> \
+               -srcstoretype jks -deststoretype pkcs12 \
+               -srcstorepass <server-jks-keystore-password> -deststorepass <server-pkcs12-keystore>
+     ```
+     
+     Example:
+     ```bash
+     $ keytool -importkeystore \
+               -srckeystore server.keystore.jks \
+               -destkeystore server.keystore.p12 \
+               -srcstoretype jks -deststoretype pkcs12 \
+               -srcstorepass changeit -deststorepass changeit
+     ```
+     
+   * Now, export the private key of the server into a `PEM` encoded file. Note that the generated `PEM` file is not protected
+     by a password. The key itself is password-protected, as we are using the `-nodes` flag. So, be sure
+     to protect it using operating system's technical and procedural controls.
+
+     ```bash
+     $ openssl pkcs12 -in <server-pkcs12-keystore-file-path> -passin pass:<key-password> \
+                      -nodes -out <server-key-file-path>
+     ```
+     
+     Example:
+     ```bash
+     $ openssl pkcs12 -in server.keystore.p12 -out server-key.key -passin pass:changeit -nodes
+     ```
+
+Step 5 concludes this stage and the stage is now set for installing the certificates and other TLS artifacts in Pravega.
+
+The table below lists the key output of this stage. 
+
+| File | Description| Command for Inspecting the Contents |
+|:-----:| :---:|:--------|
+| `server-cert.crt` | PEM-encoded CA-signed server certificate file | `openssl x509 -in server-cert.crt -text -noout`|
+| `server-key.key` | PEM-encoded file containing the server's encrypted private key |  `openssl pkcs8 -inform PEM -in server-key.key -topk8` |
+| `server.keystore.jks` | The server keystore file in .jks format | `keytool -list -v -keystore server.keystore.jks -storepass 1111_aaaa` |
```

Comment:
Refer to server keystore as "encrypted."

`-storepass` should be `changeit` instead of `1111_aaaa`

Reply:
Nice catch! :) Fixed.